frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java

```diff
@@ -67,14 +66,17 @@ public static void saveDocument(PsiFile file) {
     }
   }
 
-  public static boolean macheteFileIsOpenedAndFocused(Project project, Path macheteFilePath) {
+  public static @Nullable VirtualFile getMacheteVirtualFileIfSelected(Project project) {
     val fileEditorManager = FileEditorManager.getInstance(project);
-    val macheteVirtualFile = List.of(fileEditorManager.getSelectedFiles())
-        .find(virtualFile -> virtualFile.getPath().equals(macheteFilePath.toString()));
-    if (macheteVirtualFile.isEmpty()) {
-      return false;
-    } else {
-      return fileEditorManager.getAllEditors(macheteVirtualFile.get()).length > 0;
```

Comment:
I've checked experimentally... it looks that a selected file will always have at least editor present (that would intuitively make sense, at least)